modules/account-lib/src/mpc/curves/ed25519.ts

```diff
@@ -61,7 +61,8 @@ export class Ed25519Curve implements BaseCurve {
     );
   }
 
-  verify(y: bigint, signedMessage: Buffer): Buffer {
-    return Buffer.from(sodium.crypto_sign_open(signedMessage, bigIntToBufferLE(y, 32)));
+  verify(message: Buffer, signature: Buffer, publicKey: bigint): Buffer {
```

Comment:
Why return a buffer? What do the return values mean? Please add a doc comment if it's not going to return a boolean.

Reply:
From the docs I looked up and reading through testcases, `sodium.crypto_sign_open` returns the original message if the signature is valid. Boolean would have been ideal but I have added a comment to give more clarity on what is being returned. Seems like this is already being utilised and changing to boolean would be a breaking change now. @tylerlevine

Reply:
@joey-bai just want to bring your attention to this change to the verify function to have it follow same base parameters as ECDSA to have the message also passed to verify after discussing with @johnoliverdriscoll , I have a feeling that this could break some places, double checking with you. I think it would make sense to have it return as a boolean value if it verifies the signature correctly if there is no specific reason to return the actual message if signature is valid.

Reply:
Makes sense - i agree that boolean makes it more more clear how to use the output of this func

Reply:
updated to have it as boolean